File: Python/Ex17.py
v1 = float(input("Digite o valor 1: "))
v2 = float(input("Digite o valor 2: "))
v3 = float(input("Digite o valor 3: "))
v1 = v1 * v1
v2 = v2 * v2
v3 = v3 * v3
 
if (v1 == (v2+v3)):
        print("Esses valores formam um triângulo.")
elif(v2 == (v1+v3)):
    print("Esses valores formam um triângulo. ")
elif(v3 == (v1 + v2)):
    print("Esses valores formam um triângulo. ")
else:
    print("Esses valores não formam um triângulo. ")

File: C/Ex17.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){

  int a, b, c;

  printf("Digite o primeiro valor: ");
  scanf("%i", &a);

  printf("Digite o segundo valor: ");
  scanf("%i", &b);

  printf("Digite o terceiro valor: ");
  scanf("%i", &c);

  a = a * a;
  b = b * b;
  c = c * c;

  if(a == b + c){
      printf("Esses valores formam um triangulo.");
  }  
  else if(b == a + c){
      printf("Esses valores formam um triangulo.");
  } 
  else if(c == a + b){
      printf("Esses valores formam um triangulo.");
  } 
  else
    printf("Esses valores nao formam um triangulo.");

}